src/skolepenger/steg/5-aktivitet/UtdanningSituasjon.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+import { useLocation } from 'react-router-dom';
+import { erAllUtdanningFerdigUtfylt } from '../../../helpers/steg/aktivitetvalidering';
+import Side, { ESide } from '../../../components/side/Side';
+import { RoutesSkolepenger } from '../../routing/routes';
+import { hentPathSkolepengerOppsummering } from '../../utils';
+import { IDetaljertUtdanning } from '../../models/detaljertUtdanning';
+import { useSkolepengerSøknad } from '../../SkolepengerContext';
+import UnderUtdanning from '../../../søknad/steg/5-aktivitet/underUtdanning/UnderUtdanning';
+
+const UtdanningSituasjon: React.FC = () => {
+  const intl = useIntl();
+  const { søknad, settSøknad, mellomlagreSkolepenger } = useSkolepengerSøknad();
+  const location = useLocation();
+  const kommerFraOppsummering = location.state?.kommerFraOppsummering;
+  const skalViseKnapper = !kommerFraOppsummering
+    ? ESide.visTilbakeNesteAvbrytKnapp
+    : ESide.visTilbakeTilOppsummeringKnapp;
+
+  const oppdaterUnderUtdanning = (underUtdanning: IDetaljertUtdanning) => {
+    settSøknad((prevSøknad) => {
+      return { ...prevSøknad, utdanning: underUtdanning };
+    });
+  };
+
+  const erSisteSpørsmålBesvartOgMinstEttAlternativValgt = erAllUtdanningFerdigUtfylt(
+    søknad.utdanning
+  );
+
+  return (
+    <Side
+      tittel={intl.formatMessage({ id: 'stegtittel.arbeidssituasjon' })}
```

Comment:
Side.tsx gir oss tittel på selve steget, mens UnderUtdanning gir oss en undertittel. I dette tilfellet får vi to titler over hverandre. Bør finne på no lurt her. kanskje tittel-props til Side.tsx kan være optional?  

<img width="851" alt="Skjermbilde 2020-08-19 kl  16 21 33" src="https://user-images.githubusercontent.com/8249453/90647199-5cba9880-e238-11ea-8470-0544ee08c5cc.png">

Reply:
Enig i at det blir litt smør på flesk her. Er jo egentlig "størrelsen" til Side-tittelen vi ønsker?

Reply:
Yup, tror det. Kan sikkert gjøre tittel optional elns i en av komponentene vi gjenbruker? Litt hacky but oh well 🙈